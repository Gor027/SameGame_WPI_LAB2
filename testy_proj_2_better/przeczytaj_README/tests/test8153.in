  kcccck ck c cc kk c ck cckckc k
 k  cc kcccck   kc kkkkccc ccc   ckkkk k 
 k  cc   ckcccc kk c cck    k  c k ck  c  ckckkkkckkk cck cck ccccc c kk  c  kck
