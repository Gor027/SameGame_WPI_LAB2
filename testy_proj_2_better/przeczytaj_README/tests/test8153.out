 cc   ck c    k    ck    c
 cc k ccck c kc kckkc c kck   c kk k
cccckcckccccckkkckcckcccckccckkkckkcckckkkkckkkcckcckcccccckkckck
